packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -110,6 +110,11 @@ class CupertinoButton extends StatefulWidget {
   /// Defaults to round corners of 8 logical pixels.
   final BorderRadius borderRadius;
 
+  /// The shape of the button.
+  ///
+  /// Defaults to a super ellipse with
+//  final ShapeBorder shape;
+
```

Comment:
@jslavitz looks like this bit was checked in by mistake -- please take a moment to clean it. Thanks!